project-base/tests/ShopBundle/Database/EntityExtension/EntityExtensionTest.php

```diff
@@ -487,4 +495,88 @@ private function getOrderItem(int $id): ExtendedOrderItem
         $this->assertInstanceOf(ExtendedOrderItem::class, $result);
         return $result;
     }
+
+    private function doTestExtendedProductInstantiation(): void
+    {
+        $repository = $this->em->getRepository(Product::class);
+        $this->assertInstanceOf(ExtendedProduct::class, $repository->find(self::MAIN_PRODUCT_ID));
+
+        $query = $this->em->createQuery('SELECT p FROM ' . Product::class . ' p WHERE p.id = :id')
+            ->setParameter('id', self::MAIN_PRODUCT_ID);
+        $this->assertInstanceOf(ExtendedProduct::class, $query->getSingleResult());
+
+        $qb = $this->em->createQueryBuilder();
```

Comment:
In other test methods in this class it was shortened to `$qb`. I feel that `$qb` and `$em` are pretty common and sometime use them. Do you think that it would be more readable if the acronym was expanded?